src/components/KTooltip/KTooltip.vue

```diff
@@ -53,11 +53,12 @@ const {
   tooltipId = '',
   zIndex = 9999,
   kpopAttributes = {},
+  disabled = false,
```

Comment:
```suggestion
  disabled,
```

Our current convention is to omit `false` default value for boolean props as Vue will automatically coerce `undefined` to `false`.

Reply:
Fixed